docs/source/guides/logging.rst

```diff
@@ -2,94 +2,92 @@
+throughout the call stack to provide for maximum debuggability. Propogation of the OpenTelemetry
+parameters follows the standard provided
+`here <https://opentelemetry.lightstep.com/core-concepts/context-propagation/>`_
+
+The following are parameters which are provided in the logs as well for correlation back to
+particular requests.
+
+- `trace` is the id of a trace which tracks “the progression of a single request, as it is handled
+    by services that make up an application” -
+    `OpenTelemetry Basic Documentation <https://www.dynatrace.com/support/help/extend-dynatrace/opentelemetry/basics>`_
+- `span is` the id of a span which is contained within a trace. “A span is the building block of a
+    trace and is a named, timed operation that represents a piece of the workflow in the distributed
+    system. Multiple spans are pieced together to create a trace.” -
+    `OpenTelemetry Span Documentation <https://opentelemetry.lightstep.com/spans/>`_
+- `sampled is` the number of times this trace has been sampled. “Sampling is a mechanism to control
+    the noise and overhead introduced by OpenTelemetry by reducing the number of samples of traces
+    collected and sent to the backend.” -
+    `OpenTelemetry SDK Documentation <https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md>`_
 
 Logging Configuration
 ---------------------
 
-Logs can be configured by setting the appropriate flags in the bento configuration file for both web requests and model serving requests. Read more about how to use a bento configuration file here in the - :ref:`Configuration Guide <configuration-page>`
+Access logs can be configured by setting the appropriate flags in the bento configuration file for
+both web requests and model serving requests. Read more about how to use a bento configuration file
+here in the - :ref:`Configuration Guide <configuration-page>`
+
+To configure other logs, use the
+`default python logging configuration <https://docs.python.org/3/howto/logging.html>`_. All BentoML
+logs are logged under the ``"bentoml"`` namespace.
 
 Web Service Request Logging
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-For web requests, logging can be enabled and disabled using the `logging.access` parameter at the top level of the bentofile.yaml.
+For web requests, logging can be enabled and disabled using the `logging.access` parameter at the
+top level of the bentofile.yaml.
 
 .. code-block:: yaml
 
     logging:
       access:
           enabled: False
+          # whether to log the size of the request body
           request_content_length: True
+          # whether to log the content type of the request
           request_content_type: True
+          # whether to log the content length of the response
           response_content_length: True
+          # whether to log the content type of the response
           response_content_type: True
```

Comment:
Having the comments next to the field makes it more readable imo
```suggestion
          request_content_length: True     # whether to log the size of the request body
          request_content_type: True        # whether to log the content type of the request
          response_content_length: True  # whether to log the content length of the response
          response_content_type: True     # whether to log the content type of the response
```

Reply:
I think this would cause line wrapping, especially in small (e.g. mobile) browsers.

Reply:
I'm open to changing the format though.